kafka/rdkafka/consumer.py

```diff
@@ -20,19 +21,27 @@
 
 class SimpleConsumer(base.BaseSimpleConsumer):
 
-    def __init__(self, client, topic, partitions=None):
-        if isinstance(topic, basestring):
-            topic = client[topic]
-        self._topic = topic
-        self._partitions = partitions or self.topic.partitions # ie all
+    def __init__(self, *args, **kwargs):
+        """ For argspec see base.BaseSimpleConsumer.__init__ """
```

Comment:
IIRC, if you don't have a docstring at all, Sphinx will just use the one from the base class.